Without this patch python2.6 from host could be preferred over python from native sysroot.

Upstream-Status: Pending
diff -uNr gpsd-2.95.orig/configure.ac gpsd-2.95/configure.ac
--- gpsd-2.95.orig/configure.ac	2011-10-31 18:26:08.860719033 +0100
+++ gpsd-2.95/configure.ac	2011-10-31 18:27:47.718716963 +0100
@@ -31,7 +31,7 @@
 dnl AM_PATH_PYTHON provided with automake can be too old.  Look
 dnl for newer python first, and include 2.6 on the list.
 m4_define([_AM_PYTHON_INTERPRETER_LIST],
-	    [python2.6 python2.5 python2.4 python])
+	    [python2.7 python2.6 python2.5 python2.4 python])
 AM_PATH_PYTHON([2.4])
 ac_python=yes
 if test "x$PYTHON" = "x"; then
